apps/condo/domains/organization/access/SendNewBillingReceiptFilesNotificationsService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice organization.SendNewBillingReceiptFilesNotificationsService '--type=mutations'`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canSendNewBillingReceiptFilesNotifications ({ authentication: { item: user } }) {
```

Comment:
It's not a part of organization domain (or organization service). Move it to billing